src/main/java/org/junit/internal/runners/statements/RunBefores.java

```diff
@@ -20,6 +31,9 @@ public RunBefores(Statement next, List<FrameworkMethod> befores, Object target)
 
     @Override
     public void evaluate() throws Throwable {
+        if (null != previous) {
+            previous.evaluate();
```

Comment:
I agree, the RunBefores class suggests a different behavior. My idea is to create a class like RunIgnores, which should execute the RunBefores as next statement.